ui/components/app/approve-content-card/approve-content-card.js

```diff
@@ -73,24 +73,19 @@ export default function ApproveContentCard({
                 marginLeft={4}
                 className="approve-content-card-container__card-header__title"
               >
-                <Typography
-                  variant={TypographyVariant.H6}
-                  fontWeight={FONT_WEIGHT.BOLD}
-                >
-                  {title}
-                </Typography>
+                <Text variant={TextVariant.bodySmBold}>{title}</Text>
```

Comment:
`fontWeight={FONT_WEIGHT.BOLD}` can be removed because `TextVariant.bodySmBold` is bold